src/Programs/Programs.ts

```diff
@@ -222,9 +222,34 @@ export const Programs: Record<CompletedProgramName, Program> = {
         return;
       }
 
-      Terminal.print(targetServer.hostname + ":");
+      if (targetServer.hasAdminRights) {
```

Comment:
Admin rights are not relevant to hacking. We should leave this out, to avoid furthering this misconception.